cmd/mirror-url.go

```diff
@@ -110,21 +116,23 @@ func deltaSourceTarget(sourceURL string, targetURL string, isForce bool, isFake
 			URLsCh <- URLs{Error: errInvalidTarget(diffMsg.SecondURL)}
 			continue
 		case differInSize, differInTime:
-			if !isForce && !isFake {
-				// Size differs and force not set
-				URLsCh <- URLs{Error: errOverWriteNotAllowed(diffMsg.SecondURL)}
-				continue
-			}
-			sourceSuffix := strings.TrimPrefix(diffMsg.FirstURL, sourceURL)
-			// Either available only in source or size differs and force is set
-			targetPath := urlJoinPath(targetURL, sourceSuffix)
-			sourceContent := diffMsg.firstContent
-			targetContent := &clientContent{URL: *newClientURL(targetPath)}
-			URLsCh <- URLs{
-				SourceAlias:   sourceAlias,
-				SourceContent: sourceContent,
-				TargetAlias:   targetAlias,
-				TargetContent: targetContent,
+			if isOverwrite {
+				if !isFake {
+					// Size differs and force not set
+					URLsCh <- URLs{Error: errOverWriteNotAllowed(diffMsg.SecondURL)}
```

Comment:
Here, `isOverwrite` is enabled, we should not throw an error here.

Reply:
Again logic error.